lttoolbox/trans_exe.cc

```diff
@@ -18,6 +18,7 @@
 #include <lttoolbox/trans_exe.h>
 #include <lttoolbox/compression.h>
 #include <lttoolbox/my_stdio.h>
+#include <cstring>
```

Comment:
For what?

Reply:
there's a call to `strncmp` somewhere in the file that caused an error when I changed the includes somewhere else.